cmd/crc/cmd/config/config.go

```diff
@@ -50,8 +50,7 @@ var (
 )
 
 func init() {
-	configCmd.Long = `Modifies crc configuration properties.
-Configurable properties (enter after SUBCOMMAND): ` + "\n\n" + configurableFields()
```

Comment:
`configurableFields` was called on init() before the `defaultConfig` object was created. This triggers a panic(). 
In fact, the 2 lines are not needed at all because Long is redefined just after here: https://github.com/code-ready/crc/blob/a47969024410f4cb15287bde72294aca21c413f0/cmd/crc/cmd/config/config.go#L96